src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/WebJira.kt

```diff
@@ -94,6 +94,20 @@ data class WebJira(
         return driver.findElement(By.id("footer-build-information")).text
     }
 
+    fun goToFocusedComment(
```

Comment:
Nit: How about `goToComment`? I believe the meaning is the same and it's simpler.

Reply:
I'd consider following similar naming of either "comment" or "issue comment" instead of "focused comment" in other places of the code

Reply:
You are right about simplifying those names. I'll change it.